test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -165,6 +168,59 @@ func describeQatDpdkPlugin() {
 			ginkgo.It("does nothing", func() {})
 		})
 	})
+
+	ginkgo.Context("When a QAT device goes unresponsive", func() {
+		ginkgo.When("QAT's auto-reset is off", func() {
+			ginkgo.BeforeEach(func() {
+				ginkgo.By("creating a configMap before plugin gets deployed")
+				e2ekubectl.RunKubectlOrDie(f.Namespace.Name, "create", "configmap", "--from-literal", "qat.conf=$'ServiceEnabled=dc\nAutoresetEnabled=off", "qat-config")
+
+				ginkgo.By("setting resourceName for dc services")
+				resourceName = dcResource
+			})
+
+			ginkgo.It("checks if the heartbeat is read correctly [Functionality:heartbeat]", func(ctx context.Context) {
+				injectError(ctx, f, resourceName)
+
+				ginkgo.By("seeing if there is zero resource")
+				if err := utils.WaitForNodesWithResource(ctx, f.ClientSet, resourceName, 100*time.Second, utils.WaitForZeroResource); err != nil {
+					framework.Failf("unable to wait for nodes to have no resource: %v", err)
+				}
+			})
+		})
+
+		ginkgo.When("QAT's autoreset is on", func() {
+			ginkgo.BeforeEach(func() {
+				ginkgo.By("creating a configMap before plugin gets deployed")
+				e2ekubectl.RunKubectlOrDie(f.Namespace.Name, "create", "configmap", "--from-literal", "qat.conf=$'ServiceEnabled=dc\nAutoresetEnabled=on", "qat-config")
+
+				ginkgo.By("setting resourceName for dc services")
+				resourceName = dcResource
+			})
+
+			ginkgo.It("checks if an injected error gets solved [Functionality:auto-reset]", func(ctx context.Context) {
+				injectError(ctx, f, resourceName)
+
+				ginkgo.By("seeing if there is zero resource")
+				if err := utils.WaitForNodesWithResource(ctx, f.ClientSet, resourceName, 100*time.Second, utils.WaitForZeroResource); err != nil {
```

Comment:
Same comment about the timeout.

Reply:
Actually, yeah, true. For noResource, it does not take long time. Let me change both back to 30 seconds.

But for checking waiting postiveResource after auto_reset works, it takes some time.
So, it would be good to keep as 300 seconds (though you did not comment about that).